src/components/PageTitleJumbo.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
```

Comment:
the only difference between this component and `PageTitle` is the use of `Subtitle` versus the use of an `H2`, therefore this does not need to recreate the component. 

```
type Size = "large" | "medium";

interface Props {
  loading: boolean;
  hasData: boolean;
  title: string;
  badge: JSX.Element;
  buttons?: JSX.Element;
  size?: Size;
}

interface TitleTypographyProps {
  size: Size;
}

const TitleTypography: React.FC<TitleTypographyProps> = ({
  children,
  size,
}) => {
  if (size === "large") {
    return <H2>{children}</H2>;
  } else if (size === "medium") {
    return <Subtitle>{children}</Subtitle>;
  }
};

export const PageTitle: React.FC<Props> = ({
  loading,
  hasData,
  title,
  badge,
  buttons,
  size = "medium",
}) => (
  <>
    {!hasData && loading && (
      <PageHeader>
        <Skeleton active paragraph={{ rows: 0 }} />
      </PageHeader>
    )}
    {hasData && !loading && (
      <PageHeader>
        <TitleWrapper>
          <TitleTypography size={size}>
            <span data-cy="page-title">
              {title}
              {"  "}
              <BadgeWrapper>{badge}</BadgeWrapper>
            </span>
          </TitleTypography>
        </TitleWrapper>
        {buttons ?? null}
      </PageHeader>
    )}
  </>
);
```

Reply:
thank you! I was wondering how I would do something like that! 
it's not only the H2 vs Subtitle that's different. The css for BadgeWrapper and for PageHeader is different too. Still worth it?

Reply:
yup, just pass a prop to the styled component that conditionally renders some css